manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -115,7 +115,8 @@ MODx.grid.Grid = function(config) {
             }
 
             config.columns.push({
-                width: config.actionsColumnWidth || defaultActionsColumnWidth
+                id: config.columns.length
```

Comment:
```suggestion
                id: 'modx-actions'
```

Reply:
`modx-actions` fails the regex test from ExtJS, the `column.length` matches ids of the other header ids, don't see a reason why this should be a different

Reply:
See below (meant to reply here but managed to do so below ... haven't had enough coffee yet this morning!)